index.d.ts

```diff
@@ -0,0 +1,14 @@
+/**
+Convert an absolute path to a tilde path: `/Users/sindresorhus/dev` → `~/dev`.
+
+@example
+```
+import tildify = require('tildify');
+
+tildify('/Users/sindresorhus/dev');
+//=> '~/dev'
+```
+*/
+declare function tildify(string: string): string;
```

Comment:
We should try to do descriptive argument labels when possible.

```suggestion
declare function tildify(absolutePath: string): string;
```